### ISSUE TITLE
Staging Review finding: Unnecessary screen reader text for icons

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91571]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments FE Team
**Product:** Appointments
**Feature:** Medication Review

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When an appointment displays a Directions link with the directions icon or a phone number link with a phone icon, those icons are announced to screen reader users. This results in a repetitive experience when a screen reader announces that section of content to the user --- eg. "graphic, Directions icon, link, Directions."

I would categorize these icons as purely decorative. They're helpful for sighted users but they don't add any information not already present in the text. They don't need to be announced by screen readers. (This was previously flagged in #88450.)

<details><summary>Link, screenshot or steps to recreate</summary>

These icons are coded using the `va-icon` component, eg:
```
<va-icon icon="directions" size="3" srtext="Directions icon" class="hydrated"></va-icon>
```

</details>

## Recommended action

In the `va-icon` component, just remove the `srtext` prop entirely.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
